jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -114,27 +115,33 @@ def __init__(self, *args, **kwargs):
         self.viewer = ViewerSelect(self, 'viewer_items', 'viewer_selected', 'multiselect')
         self.layer = LayerSelect(self, 'layer_items', 'layer_selected', 'viewer_selected', 'multiselect')  # noqa
 
-        def not_profile(state):
-            return not isinstance(state, (ProfileViewerState, ProfileLayerState))
-
         def is_profile(state):
             return isinstance(state, (ProfileViewerState, ProfileLayerState))
 
+        def not_profile(state):
+            return not is_profile(state)
+
+        def is_image(state):
+            return isinstance(state, BqplotImageLayerState)
+
+        def not_image(state):
+            return not is_image(state)
+
         def is_spatial_subset(state):
             return isinstance(state, ImageSubsetLayerState)
 
         def is_not_subset(state):
-            return not isinstance(state, ImageSubsetLayerState)
+            return not is_spatial_subset(state)
 
-        # Spectrum viewer/layer options:
+        # Profile/line viewer/layer options:
         self.layer_visible = PlotOptionsSyncState(self, self.viewer, self.layer, 'visible',
                                                   'layer_visible_value', 'layer_visible_sync',
                                                   state_filter=is_not_subset)
         self.collapse_function = PlotOptionsSyncState(self, self.viewer, self.layer, 'function',
                                                       'collapse_func_value', 'collapse_func_sync')
         self.line_color = PlotOptionsSyncState(self, self.viewer, self.layer, 'color',
                                                'line_color_value', 'line_color_sync',
-                                               state_filter=is_profile)
+                                               state_filter=not_image)

```

Comment:
Lines used to just be spectra, but now we have a line that is in an image viewer and needs to expose the right plot options.